CHANGELOG.md

```diff
@@ -161,6 +161,13 @@ Other minor additions
   mapM      : (Q ⊆ M ∘′ P) → All Q ⊆ (M ∘′ All P)
   forA      : All Q xs → (Q ⊆ F ∘′ P) → F (All P xs)
   forM      : All Q xs → (Q ⊆ M ∘′ P) → M (All P xs)
+
+  _++_      : ∀ {l m} → All P l → All P m → All P (l List.++ m)
```

Comment:
Note that this one is in [`Data.List.All.Properties`](https://agda.github.io/agda-stdlib/Data.List.All.Properties.html#4773)